upup/pkg/fi/cloudup/bootstrapchannelbuilder/bootstrapchannelbuilder.go

```diff
@@ -542,6 +542,27 @@ func (b *BootstrapChannelBuilder) buildAddons(c *fi.ModelBuilderContext) (*chann
 		}
 	}
 
+	npd := b.Cluster.Spec.NodeProblemDetector
+
+	if npd != nil && fi.BoolValue(npd.Enabled) {
+
+		key := "node-problem-detector.addons.k8s.io"
+		version := "0.8.7"
+
+		{
+			location := key + "/k8s-1.19.yaml"
```

Comment:
Should be `k8s-1.17.yaml` if supported on k8s 1.17, otherwise need an API validation to enforce k8s >= 1.19.

Reply:
thanks ^